knowledge-base/tabstrip-add-remove-tabs.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Add and Remove Tabs
+description: Learn how to dynamically add and remove tabs
+type: how-to
+page_title: Add and Remove Tabs
+slug: tabstrip-kb-add-remove-tabs
+tags: telerik,blazor,tabstrip,add tabs,remove tabs
+ticketid: 
+res_type: kb
+previous_url: /knowledge-base/tabstrip-dynamic-tabs
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>TabStrip for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+I have a collection of items representing separate tabs. I am iterating through that collection to render a tab for each item as shown [here](slug:tabstrip-tabs-collection). I want to allow the user to add and remove tabs, how to achieve that?
```

Comment:
Avoiding generic words like "here" lets you improve the SEO and quality of the text.
```suggestion
I have a collection of items representing separate tabs. I am iterating through that collection to render a tab for each item as shown in the [Tabs Collection article](slug:tabstrip-tabs-collection). I want to allow the user to add and remove tabs. How to achieve that?
```